README.md

```diff
@@ -50,6 +50,23 @@ For issues see [Snovault OSX App Installation][] first.
     $ ./circle-tests.sh npm
     ```
 
+## Configuration Installation (In Progress)
+    The configuration repo is set by 'conf_dir = ./cloud-config' in the 'dev-config.ini' file.  The
+    'deploy.py' compiles yaml files from 'cloud-config' for use with aws instances.  There is a
+    very specific file structure at the moment.  We are working to generalize the use case.
+
+    To use a different directory for cloud configuration copy 'dev-config.ini' to '.dev-config.ini'.
```

Comment:
I think here it should be demo-config.ini and not dev-config.ini

Reply:
@caseylitton here is the comment in the PR I left you yesterday

Reply:
updated.